public/app/features/dashboard-scene/scene/DashboardScene.tsx

```diff
@@ -141,6 +149,7 @@ export class DashboardScene extends SceneObjectBase<DashboardSceneState> {
       editable: true,
       body: state.body ?? new SceneFlexLayout({ children: [] }),
       links: state.links ?? [],
+      copiedPanel: store.exists(LS_PANEL_COPY_KEY),
```

Comment:
not sure why we need this boolean flag in state (when it exits in local storage)